podcast/the-changelog-365.md

```diff
@@ -168,8 +168,6 @@ We could start to think about having lots of message passing between components,
 
 **Chris Bell:** Or restarts. If we've got supervisors, we've got restart strategies. So we can say "That bit of the UI will boot back up into its previous state, or some known good state."
 
-**Desmond Bowe:** \[unintelligible 00:32:52.25\]
-
```

Comment:
Why the delete? Did Desmond just not say anything substantive here?

Reply:
For me it sounds like he accidentally turned on his mic.

Reply:
Looks like someone else figured out it as "That’s true again." for this transcript.